### PR TITLE
Script for iterating development on GCE

### PR DESCRIPTION
**Pull request by justinsb**
AWS one to follow!

### CODE REVIEW

File: hack/dev-build-gce.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+bazelisk run //cmd/kops version -- --short || return
+KOPS_VERSION=$(bazelisk run //cmd/kops version -- --short)
+export KOPS_BASE_URL=https://storage.googleapis.com/${UPLOAD_DEST_BUCKET}/kops/${KOPS_VERSION}/
+
+# Create the state-store bucket if it doesn't exist
+export KOPS_STATE_STORE=gs://kops-state-$(gcloud config get-value project)
+gsutil ls "${KOPS_STATE_STORE}" || gsutil mb "${KOPS_STATE_STORE}" || return
+
+# Set feature flags needed on GCE
+export KOPS_FEATURE_FLAGS=AlphaAllowGCE
```
Comment (zetaab) [inline hack/dev-build-gce.sh:43]: is this really needed nowadays?

Comment (justinsb) [reply hack/dev-build-gce.sh:43]: Yes, but I'd like to fix that :-)

File: hack/dev-build-gce.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# Can't use set -e in a script we want to source
+#set -e
+
+#set -x
+
+# Build and upload to bucket
+UPLOAD_DEST_BUCKET="kops-dev-$(gcloud config get-value project)-${USER}"
+export UPLOAD_DEST=gs://${UPLOAD_DEST_BUCKET}
+gsutil ls "${UPLOAD_DEST}" || gsutil mb "${UPLOAD_DEST}" || return
+make kops-install dev-upload || return
```
Comment (johngmyers) [inline hack/dev-build-gce.sh:31]: I find using dev-upload-linux-amd64 saves a lot of time. dev-upload will rebuild and re-upload from scratch on every run.

Comment (justinsb) [reply hack/dev-build-gce.sh]: Good tip - changed, and also made sure to set KOPS_ARCH to only be amd64.

**Review by johngmyers — APPROVED** [review]

**Pull request merged**